### ISSUE TITLE
Output from MAPIR plugin is green

### ISSUE THREAD

**Issue by dmalloy** [mapircamera/QGIS#12]
I am trying to process 56 photos shot with the RGB Survey 2 camera and all of the processed pictures (resulting tiff images) have a green color to them. Original JPEGs that were geotagged and used in this test were fine. I have tried using different filters and different versions of QGIS and the MAPIR plugin. My last attempt was with the latest version of both (QGIS 2.18.9 and MAPIR 1.13). This was a test for creating a basic ortho mosaic. Any idea why the images are coming out green?

![2017_0330_000238_057](https://cloud.githubusercontent.com/assets/26972426/26607282/967c088a-454a-11e7-98fe-18dc2b2fc3d1.png)

![2017_0330_000240_058](https://cloud.githubusercontent.com/assets/26972426/26607085/93d924ba-4549-11e7-9fac-1e1ac02cab5d.JPG)

**Comment by mapircamera** [comment]
Yes, due to the nature of our sensor/filter combination, the images can be saturated with green light. We are trying to find a solution that does not effect the histogram of the data or the resolution of the image. Because most of our customers are surveyors, the plugin tries to keep the data as untouched as possible. At one point we had attempted some color correction, but we have had little luck solving the problem in a programmatic fashion.
